packages/gatsby-plugin-sass/README.md

```diff
@@ -43,7 +43,13 @@ and common cross-browser flexbox bugs. Normally you don't need to think about it
 you'd prefer to add additional postprocessing to your SASS output you can sepecify plugins
 in the plugin options
 
-## Other Options
+## Known issues
```

Comment:
I wouldn't title this "known issues" since it's not clear that it's an issue per se as this is a sass convention.

Perhaps title the section "Relative paths & `url()`"?